src/xr/xr-finger.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @class
+ * @name pc.XrFinger
+ * @classdesc Represents finger with related joints and index
+ * @description Represents finger with related joints and index
+ * @param {number} index - Index of a finger
```

Comment:
What's the range here? 0 to 4 inclusive?

Reply:
Yes.